lints/lint_subject_multiple_rdn_test.go

```diff
@@ -21,7 +21,7 @@ import (
 func TestSubjectRDNTwoAttribute(t *testing.T) {
 	inputPath := "../testlint/testCerts/subjectRDNTwoAttribute.pem"
 	expected := Warn
```

Comment:
Shouldn't this be `Notice` instead of `Warn`? (with the same change to the implementation)

Reply:
🤦‍♂️